microcosm_flask/tests/conventions/fixtures.py

```diff
@@ -81,6 +83,13 @@ def csv_column_order(self):
         return column_order
 
 
+@add_associated_schema(
+    "Foo",
```

Comment:
I'd give this a meaningful name otherwise the intent is confusing (I don't understand what it is). Is it a "PersonSubSchema" or a "PersonPubsubSchema"?

Reply:
Maybe associatred schemas could use an extra property to specify their intent? We are basically adding a feature for the webservice to expose information about schemas that it may not use, so we may want to create a facility for those schemas to document themeslves a little more.

Reply:
Good points. I'll update the tests and see if some "docstring" can be exposed in swagger.